src/components/data_comps/docn/docn_comp_mod.F90

```diff
@@ -417,7 +416,6 @@ subroutine docn_comp_run(EClock, x2o, o2x, &
 
     lsize = mct_avect_lsize(o2x)
     do n = 1,lsize
-       o2x%rAttr(km   ,n) = imask(n)
```

Comment:
I don't think this should have been removed.

Reply:
See the comment above for lfrac.